mesh/submesh/submesh_utils.hpp

```diff
@@ -40,15 +43,6 @@ struct UniqueIndexGenerator
    int Get(int i, bool &new_index);
 };
 
-/**
- * @brief Given an element @a el and a list of @a attributes, determine if that
- * element is in at least one attribute of @a attributes.
- *
- * @param el The element
- * @param attributes The attributes
- */
-bool ElementHasAttribute(const Element &el, const Array<int> &attributes);
```

Comment:
Can this be kept for backwards compatibility? Even if it is not used anywhere in MFEM, is it possible that an MFEM application uses it?

Reply:
I would posit that this isn't really supposed to be public api, but can easily add:
```
bool ElementHasAttribute(const Element &el, const Array<int> &attributes)
{
   return HasAttribute(el,attributes);
}
```

Reply:
We usually avoid changes to the public API, even when it doesn't make sense, to avoid breaking applications that depend on MFEM. It looks your suggested change would maintain the API.